pyanaconda/ihelp.py

```diff
@@ -71,77 +117,67 @@ def _get_best_help_file(help_folder, help_file):
         log.warning("no help content found for file %s", help_file)
         return None
 
-def get_help_path(help_file, instclass, plain_text=False):
-    """
-    Return the full path for the given help file name,
-    if the help file path does not exist a fallback path is returned.
-    There are actually two possible fallback paths that might be returned:
+def get_placeholder_path(instclass, plain_text=False):
+    """Get path to appropriate placeholder for given install class.
 
-    * first we try to return path to the main page of the installation guide
-      (if it exists)
-    * if we can't find the main page of the installation page, path to a
-      "no help found" placeholder bundled with Anaconda is returned
+    :param instclass: currently active install class instance
+    :param str plain_text: point to plain text version of the placeholder
+    :returns: path to placeholder or None if no placeholder is found
+    :rtype: str or None
+    """
+    if plain_text:
+        placeholder = installclass.help_placeholder_plain_text
+    else:
+        placeholder = instclass.help_placeholder
+    placeholder_path = _get_best_help_file(instclass.help_folder, placeholder)
+    if placeholder_path:
+        return placeholder_path
+    else:
+        log.error("placeholder {placeholder_name} not found in {help_folder}".format(placeholder_name=instclass.help_placeholder,
+                                                                                     help_folder=instclass.help_folder))
```

Comment:
In this particular case I would prefer to use just `{}`  instead of the long name. It is not used for translation and it should be able to get to one line, thought.